### PR TITLE
Show a warning if case config file has missing panels, do not crash

### LINKED ISSUE #2608: Missing gene panels in yaml
A lot of frustration comes up now and then when you try to load a case and one or many of the defined gene panels are missing in the database. This could either be because of differences in databases between developmental servers and live, and also if some old gene panel were to be removed, all old yaml files would suffer from this.

Would it be possible for scout, instead of just aborting, to just throw a warning at the end. X gene panel did not exist in database. At least maybe an option to ignore non-existing gene panels in the load command?

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2608

**How to test**:
1. Edit demo case config file with a gene panel that doesn't exist (example: `gene_panels: [panel1, hello]`)
2. Load the case with `scout setup demo` command or load it manually with `scout --demo load case scout/demo/643594.config.yaml --update`
3. Master branch should crash, this branch should load the case with a warning.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR, DN

### CODE REVIEW

**Comment by northwestwitch** [comment]
Ready for review now

File: tests/build/test_build_case.py [hunk 1]
```diff
@@ -118,7 +118,13 @@ def test_build_case_non_existing_owner(adapter, institute_obj):
         case_obj = build_case(case_info, adapter)
 
 
-# def test_build_case_config(parsed_case):
-#     case_obj = build_case(parsed_case)
-#     print(case_obj.to_json())
-#     assert False
+def test_build_case_no_valid_panel(adapter, institute_obj):
```
Comment (dnil) [inline tests/build/test_build_case.py:121]: 👍

File: scout/server/blueprints/variants/controllers.py [hunk 2]
```diff
@@ -1012,12 +1012,12 @@ def check_form_gene_symbols(
             "label": "info",
         },
         "not_found_symbols": {
-            "message": "HGNC symbols not present in database genes collection",
+            "message": "HGNC symbols not present in database's genes collection",
```
Comment (dnil) [inline scout/server/blueprints/variants/controllers.py:1015]: 😸

File: scout/build/case.py [hunk 3]
```diff
@@ -169,7 +169,11 @@ def build_case(case_data, adapter):
     for panel_name in case_panels:
         panel_obj = adapter.gene_panel(panel_name)
         if not panel_obj:
-            raise IntegrityError("Panel %s does not exist in database" % panel_name)
+            LOG.warning(
```
Comment (dnil) [inline scout/build/case.py:172]: 👍

**Review by dnil — APPROVED** [review]

**Pull request merged**